.github/workflows/main.yml

```
@@ -23,29 +23,35 @@ jobs:
             TARGET: macOS
           - os: windows-latest
             TARGET: Windows
-        python-version: [ "3.10" ]
+        python-version: [ "3.12" ]
 
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)